### PR TITLE
AST-22257 - Adding support to critical vulnerabilities

### PR DESCRIPTION
**Pull request by igorlombacx**
By submitting a PR to this repository, you agree to the terms within the [Checkmarx Code of Conduct](../docs/code_of_conduct.md). Please see the [contributing guidelines](../docs/contributing.md) for how to create and submit a high-quality PR for this repo.

### Description

image

image

### References

https://checkmarx.atlassian.net/browse/AST-22257?atlOrigin=eyJpIjoiZGVjNDQ2MTZiMmViNGVhOTlhOWEwNmEwYzZmYzBkMzkiLCJwIjoiaiJ9

### Checklist

- [x] I have added documentation for new/changed functionality in this PR (if applicable).
- [x] All active GitHub checks for tests, formatting, and security are passing
- [x] The correct base branch is being used

### CODE REVIEW

File: .idea/.gitignore
Comment (AlvoBen) [inline .idea/.gitignore:1]: why pushing .idea?

File: ast-visual-studio-extension-tests/cx-unit-tests/cx-wrapper-tests/ResultTest.cs [hunk 1]
```diff
@@ -27,8 +27,9 @@ public void TestResultsJSON()
         {
             List<Scan> scanList = cxWrapper.GetScans("statuses=Completed");
             Assert.True(scanList.Any());
+            Scan scan = scanList.FirstOrDefault(scan => scan.Status.ToLower() == "completed");
```
Comment (AlvoBen) [inline ast-visual-studio-extension-tests/cx-unit-tests/cx-wrapper-tests/ResultTest.cs:30]: make "completed" constant

File: .idea/.gitignore
Comment (hmmachadocx) [reply .idea/.gitignore:1]: It was by mistake. already fixed!

File: ast-visual-studio-extension-tests/cx-unit-tests/cx-wrapper-tests/ResultTest.cs (hunk 1 above)
Comment (hmmachadocx) [reply ast-visual-studio-extension-tests/cx-unit-tests/cx-wrapper-tests/ResultTest.cs]: Done, thanks

**Review by AlvoBen — APPROVED** [review]

**Pull request merged**